chatbot/conversation_start.php

```diff
@@ -17,7 +17,7 @@
 $thisFile = __FILE__;
 
 /** @noinspection PhpIncludeInspection */
-require_once("../config/global_config.php");
+require_once(_CONF_PATH_ . 'global_config.php');
```

Comment:
The problem here is that `conversation_start.php` isn't always called from a script that has that constant defined. Bear in mind that it's also called via AJAX, so no such definition has been made yet. Your proposed change will break the jQuery GUI, so is not a workable solution.